src/pages/index.js

```diff
@@ -126,13 +126,17 @@ function getCardElement(data) {
   return cardElement;
 }
 
+function toggleLikeClass(evt) {
+  evt.target.classList.toggle("card__like-button_liked");
+}
+
 function handleCardLike(evt, dataId) {
   let isLiked = evt.target.classList.contains("card__like-button_liked")
     ? true
     : false;
   api
     .toggleLike(dataId, isLiked)
-    .then(evt.target.classList.toggle("card__like-button_liked"))
+    .then(toggleLikeClass(evt))
```

Comment:
In `then`, `catch`, `finally` you need to pass only the **reference** to the function (not a function call) otherwise the function is called immediately without waiting for the response from the server. You can pass an **unnamed** arrow function as well. 


Here will be


```js
  api
    .toggleLike(dataId, isLiked)
    .then(() => evt.target.classList.toggle("card__like-button_liked"))
    .catch(console.error);

```

Reply:
fixed